src/components/content-editor/resource-editor.tsx

```diff
@@ -172,6 +174,15 @@ class ResourceEditor extends React.Component<Props, {}> {
             onChange={this.onResourceDataChange}
           />
         );
+      case ExhibitionPageResourceType.Color:
+        return (
+          <ColorPicker
+            color={resource.data}
+            onChangeComplete={({ rgb: { r, g, b, a } }) =>
+              this.updateResourceData(`rgba(${r}, ${g}, ${b}, ${a})`)
```

Comment:
Check whether alpha can be undefined, if it can then default to 100%

Reply:
Resolved.